CHANGES/1026.bugfix.rst

```diff
@@ -0,0 +1,20 @@
+Joining URLs with empty segments has been changed
+to match :rfc:`3986`.
+
+Previously empty segments would be removed from path,
+breaking use-cases such as
+
+.. code-block:: python
+
+   URL("https://web.archive.org/web/") / "https://github.com/"
+
+Now :meth:`~yarl.URL.__truediv__` and :meth:`~yarl.URL.joinpath` operation
+keep empty segments, but do not introduce new empty segments.
+e.g.
+
+.. code-block:: python
+
+   URL("https://example.org/") / ""
+
+does not introduce an empty segment.
+-- by :user:`commonism` and :user:`youtux`.
```

Comment:
Since you're not putting this into a sentence, move it to a separate line and drop that trailing period.
```suggestion

-- by :user:`commonism` and :user:`youtux`
```

Reply:
(alternatively, you could append the byline to the very first sentence, just don't put a period before it if you decide to go that route)